pkg/machine/e2e/basic_test.go

```diff
@@ -69,6 +70,37 @@ var _ = Describe("run basic podman commands", func() {
 		Expect(rmCon).To(Exit(0))
 	})
 
+	It("Volume ops", func() {
+		skipIfVmtype(define.HyperVVirt, "Hyper-V does not have functioning volume mounts yet #21036")
+
+		// golangci-lint has trouble with actually skipping tests marked Skip
+		// so skip it on cirrus envs and where CIRRUS_CI isn't set.
+		name := randomString()
+		i := new(initMachine)
+		session, err := mb.setName(name).setCmd(i.withImage(mb.imagePath).withNow()).run()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(session).To(Exit(0))
+
+		// Create under home under the VMs system mount
+		hDir, err := os.UserHomeDir()
+		Expect(err).ToNot(HaveOccurred())
+		tDir, err := os.MkdirTemp(hDir, "attr-test")
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
Does this create the dir under my real homedir or the fake test homedir?
I think it would be safer to just create a tmpdir and mount that on machine init explicitly so that the container can use it.

Reply:
It would end up being fakeish home since CI overrides HOME, but doing it in a test  temp should work fine for the purpose of the test so posted a change up for that